docs/reference/requirement-sets/shared-runtime-requirement-sets.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Shared runtime requirement sets
+description: 'Specifies the platforms and Office hosts that support the SharedRuntime APIs.'
+ms.date: 01/21/2020
+ms.prod: non-product-specific
+localization_priority: Normal
+---
+
+# Shared runtime requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
+
+Parts of an Office add-in that run JavaScript code; including task panes, function files launched from Add-in Commands, and Excel custom functions; can share a single JavaScript runtime. This enables all the parts to share a set of global variables, to share a set of loaded libraries, and to communicate with each other without having to pass messages through a persisted storage.
+
+The following table lists the SharedRuntime 1.1 requirement set, the Office host applications that support that requirement set, and the build or version numbers for the Office application.
+
+|  Requirement set  |  Office 2013 (or later) on Windows<br>(one-time purchase) | Office on Windows<br>(connected to Office 365 subscription)   |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web  | Office Online Server |
+|:-----|:-----|:-----|:-----|:-----|:-----|:-----|
+| SharedRuntime 1.1  | N/A | Version 2001 (Build nnnnn.nnnnn) or later | N/A | nn.nn or later | January 2020 | N/A |
```

Comment:
You already mentioned this in the PR description, just flagging here as well -- re needing to update the build numbers in this table entry.